lib/gtfs_parser.rb

```diff
@@ -11,7 +11,7 @@ module GtfsParser
   GTFS_PROTOCOL = 'http://'.freeze
   GTFS_HOST = 'pvta.com'.freeze
   GTFS_PATH = '/g_trans/google_transit.zip'.freeze
-  LOG_ARCHIVES = File.expand_path('../../log_directory', __FILE__)
+  log = File.expand_path('../../log', __FILE__)
```

Comment:
Since this is a constant, it'd be nicer to have it declared in all-caps — otherwise it *looks* like an undefined local variable when it's used in methods.